### ISSUE TITLE
Difficulty serving notebooks through Mercury using ngrok

### ISSUE THREAD

**Issue by amcmax1** [mljar/mercury#255]
I'm having some difficulty accessing the notebooks using Mercury with a developer ngrok account.

1. After running 'mercury run' in the directory of the notebooks, I run the following:

`ngrok http 8000 --host-header rewrite`

(without the --host-header flag, result is the same)

2. When I use the forwarded Ngrok URL (https://5214d4130e78.ngrok.app) I get the following with the Mercury UI header: 

```
Site does not exist

We can't find site you are looking for. Please make sure that URL address is correct.
```

3. But when I use the following localhost URL, the notebooks are served to the browser just fine: http://localhost:8000/

I just assumed the whole flow would be simple (which is what I assume for ngrok).  I just upgraded for a dev Ngrok account because at first I thought it was an issue on their end, but I'm not sure any longer.

Is it an issue on Mercury's side?

**Comment by pplonski** [comment]
Hi @amcmax1,

yes, that's might be the issue with Mercury ...

Mercury is trying to guess the server address based on frontend URL address. In this case, it is trying to connect to the server at `ngrok.app`. I will fix this tomorrow.

BTW, are you happy with web app created from notebook with Mercury?

**Comment by amcmax1** [comment]
Yes, otherwise everything's great.  💯 
Looking forward to share some notebooks with my team (without deploying to a cloud server) using Mercury 😄

**Comment by pplonski** [comment]
Hi @amcmax1,

I've released version `2.2.8` that fixes the problem with forwarding. It should be working :)

New version is already in PyPi, should be soon on conda-forge channel.

![Peek 2023-04-27 18-10](https://user-images.githubusercontent.com/6959032/234923633-0fb50115-41d2-4e1b-9929-e5c002103ff3.gif)
